include/sys/file.h

```diff
@@ -11,7 +11,9 @@
 
 #include "zos-macros.h"
 
-#if (__EDC_TARGET < 0x42050000)
+#if (__TARGET_LIB__ < 0x42050000) || \
+    (__TARGET_LIB__ >= 0x42050000 && \
+    (((_POSIX_C_SOURCE + 0) < 200809L) && !defined(_XPLATFORM_SOURCE)))
```

Comment:
```suggestion
#if (__TARGET_LIB__ < 0x42050000) || \
    (((_POSIX_C_SOURCE + 0) < 200809L) && !defined(_XPLATFORM_SOURCE))
```
If a is not < b then a >= b. Same applies to the change in zos-getentropy.h (and a few other places).